### PR TITLE
314 dynamic intakes on pages

### PR DESCRIPTION
**Pull request by AntBush**
### Description: 

PR to add intake dates that match the real intake dates in the DB.

References 314

### Type of change:

- [X] New feature (non-breaking change which adds functionality)

## Acceptance criteria

- [ ] Show the intake starting date: September 7, 2022 9:00AM PT
- [x] Show the intake end date: November 6, 2022 9:00AM PT

## Pages affected
- [x] Landing page 
- Current Intake
- [x] Dashboard page 
- Current Intake
- [x] Submission page 
- Draft Status = Current Intake
- Submit Status = My intake
- [x] Submission success page 
- Submit Status = My intake

### Checklist for DoD:
- [x]  Code commented, checked in and ran against current version in source control 
- [x]  Obey “Conventional commits” (https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Code review (https://google.github.io/eng-practices/review/)
- [x]  Peer reviewed (or produced with pair programming) and meeting development standards ([Google](https://google.github.io/eng-practices/review/reviewer/)) 
- [x]  Unit tests are written and passing 
- [x]  Don’t introduce breaking changes. 
- [x]  Don’t (avoid) introducing more tech debt. 
- [x]  Automatic Tests implemented and running 

### Additional Notes:

_(Include any notes you have about this pull request)_

### CODE REVIEW

**Review by marcellmueller — COMMENTED** [review]
Looks good and is working well, just wondering about the missing acceptance criteria for the intake opening date. I noticed currently it says the opening date is August 19th, is that something we are updating in a separate PR?

File: app/pages/index.tsx [hunk 1]
```diff
@@ -25,12 +26,20 @@ const getPagesQuery = graphql`
...
 `;
 
 // eslint-disable-next-line @typescript-eslint/ban-types
 const Home = ({ preloadedQuery }: RelayProps<{}, pagesQuery>) => {
-  const { session } = usePreloadedQuery(getPagesQuery, preloadedQuery);
+  const { session, openIntake } = usePreloadedQuery(
+    getPagesQuery,
+    preloadedQuery
+  );
+  console.log(openIntake)
```
Comment (marcellmueller) [inline app/pages/index.tsx:42]: this lil guy snuck in here :sweat_smile:

**Comment by AntBush** [comment]
I think the idea is to later update the intake open date, although thinking about it now I could just hardcode it in. Mars mentioned that it wouldn't be an issue and that in the future when the 'Time Travel' is implemented we'd change the intake date to the correct date

File: app/pages/index.tsx (hunk 1 above)
Comment (AntBush) [reply app/pages/index.tsx]: My achilles heel! Pushed a commit to fix

**Comment by marcellmueller** [comment]
> I think the idea is to later update the intake open date

Yeah I figured so, especially since it might change.

**Comment by eazeredo** [comment]
> the 'Time Travel' is implemented we'd change the intake date to the correct date

The time travel will not change the intake dates. It will allow emulating the system date (today/now) to a specific date in the past or in the future. @AntBush @marcellmueller

**Comment by AntBush** [comment]
>The time travel will not change the intake dates

I meant moreso that, when the 'Time Travel' is implemented we can change the start date to the real date without breaking functionality (by using time travel).

**Comment by eazeredo** [comment]
Nope, the only thing the time travel will change is the system date. Nothing in the database will change.

It will be us going to the future, not the application going back to the past.

**Pull request merged**